akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -468,57 +469,101 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
   /**
    * INTERNAL API
    */
-  private[cluster] class ClusterLogger(log: LoggingAdapter) {
+  private[cluster] class ClusterLogger(log: MarkerLoggingAdapter) {
     def isDebugEnabled: Boolean =
       log.isDebugEnabled
 
     def logDebug(message: String): Unit =
-      logAtLevel(Logging.DebugLevel, message)
+      if (LogInfo && log.isDebugEnabled)
```

Comment:
Hmm, why check this here and not inside `logAtLevel`?

Reply:
would be fine here but for the others with replacement arguments I want the `if` before `log.format`

Reply:
Aha, that explains it, I missed out on that, so NVM